protected/controllers/accessRequest/Respond.php

```diff
@@ -0,0 +1,53 @@
+<?php
+declare(strict_types=1);
+
+namespace prime\controllers\accessRequest;
+
+use prime\components\NotificationService;
+use prime\interfaces\AccessCheckInterface;
+use prime\models\ar\AccessRequest;
+use prime\models\ar\Permission;
+use prime\models\forms\accessRequest\Respond as RespondFormModel;
+use SamIT\abac\AuthManager;
+use yii\base\Action;
+use yii\mail\MailerInterface;
+use yii\web\Request;
+use yii\web\User as UserComponent;
+
+class Respond extends Action
+{
+    public function run(
+        Request $request,
+        UserComponent $user,
+        AuthManager $abacManager,
+        AccessCheckInterface $accessCheck,
+        NotificationService $notificationService,
+        int $id
+    ) {
+        $accessRequest = AccessRequest::findOne(['id' => $id]);
+
+        $accessCheck->requirePermission($accessRequest, Permission::PERMISSION_RESPOND);
+        $model = \Yii::createObject(RespondFormModel::class, [
```

Comment:
Don't use `Yii::createObject()`. Either create the object directly or create a factory and inject it.